docs/docs/ProgrammingGuide/run-on-dataproc.md

```diff
@@ -1,52 +1,106 @@
+
+
+You can run BigDL examples using the `run.example.sh` script with following parameters:
+
+* Mandatory parameters:
+  
+    * `-m|--model` which model to train, including
+    
+        * lenet: train the [LeNet](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/lenet) example
+    
+        * vgg: train the [VGG](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/vgg) example
+
+        * inception-v1: train the [Inception v1](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception) example
+
+        * perf: test the training speed using the [Inception v1](https://github.com/intel-analytics/BigDL/blob/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/Inception_v1.scala) model with dummy data
+
+    * `-s|--spark-url` the master URL for the Spark cluster
+
+    * `-n|--nodes` number of Spark slave nodes
+
+    * `-o|--cores` number of cores used on each node
 
-* Jupyter Notebook: [http://bigdl-m:8888/](http://bigdl-m:8888/)
+    * `-r|--memory` memory used on each node, e.g. 200g
 
-* Tensorboard: [http://bigdl-m:6006/](http://bigdl-m:6006/)
+    * `-b|--batch-size` batch size when training the model; it is expected to be a multiple of "nodes * cores"
 
-* YARN ResourceManager: [http://bigdl-m:8088/](http://bigdl-m:8088/)
+    * `-f|--hdfs-data-dir` HDFS directory for the input images (for the "inception-v1" model training only)
 
-Inside your Jupyter Notebook, you may find two examples are already there. Start your BigDL journey with them.
+* Optional parameters:
+
+    * `-e|--max-epoch` the maximum number of epochs (i.e., going through all the input data once) used in the training; default to 90 if not specified
+
+    * `-p|--spark` by default the example will run with Spark 1.5 or 1.6; to use Spark 2.0, please specify "spark_2.0" here (it is highly recommended to use _**Java 8**_ when running BigDL for Spark 2.0, otherwise you may observe very poor performance)
+
+    * `-l|--learning-rate` by default the the example will use an initial learning rate of "0.01"; you can specify a different value here
+
+After the training, you can check the log files and generated models  
+
+Replace $BIGDLJAR with bigdl binary name in ./lib in below command, eg: bigdl-SPARK_2.2-0.3.0-jar-with-dependencies.jar  
+
+eg: ./bin/run.example.sh --model lenet --nodes 2 --cores 2 --memory 1g --batch-size 16 -j lib/$BIGDLJAR -p spark_buildIn
+
+You can also run lenet examples in below command:   
+```bash
+spark-submit \
+ --executor-cores 2 \
+ --num-executors 2 \
+ --driver-class-path ./lib/$BIGDLJAR \
+ --class com.intel.analytics.bigdl.models.lenet.Train \
+ ./lib/$BIGDLJAR \
+ -f ./mnist \
+ -b 16
+```
+---
+## **Run BigDL Python example**
+Download lenet5.py from https://github.com/intel-analytics/BigDL/blob/master/pyspark/bigdl/models/lenet/lenet5.py
```

Comment:
Isn't this file contained in the BigDL package just downloaded?

Reply:
BigDL package includes some binaries and scripts, python files are not included.